gloo/docs/gloo_routing/virtual_services/security/custom_auth/_index.md

```diff
@@ -65,111 +102,122 @@ if __name__ == "__main__":
 
 As you can see, this service will allow requests to `/api/pets/1` and will deny everything else.
 
-{{% notice tip %}}
-You can easily change the sample auth server. When using minikube, download the [Dockerfile](Dockerfile) and the [server code](server.py) and just run:
-
-```shell
-eval $(minikube docker-env)
-docker build -t quay.io/solo-io/sample-auth .
-kubectl --namespace gloo-system delete pod -l app=sample-auth
-```
-{{% /notice %}}
-
 ### Deploy auth service
 
-To add this service to your cluster, download the [auth-service yaml](auth-service.yaml) and apply it:
+To deploy this service to your cluster, download the [auth-service yaml](auth-service.yaml) file and apply it:
 
 ```shell
 kubectl apply --filename auth-service.yaml
 ```
 
 This file contains the deployment, service and upstream definitions.
 
+{{% notice tip %}}
+When running in `minikube` you can easily update this sample auth service. Just download the [Dockerfile](Dockerfile) and 
+the [server.py file](server.py) to a local directory, apply your changes to the server code, and run the following commands:
+
+```shell
+eval $(minikube docker-env)
+docker build -t quay.io/solo-io/sample-auth .
+kubectl --namespace gloo-system delete pod -l app=sample-auth
+```
+{{% /notice %}}
+
 ## Configure Gloo to use your server
 
 ### Configure Gloo settings
 
-Edit the gloo settings (`kubectl --namespace gloo-system edit settings default`) to point to your auth server.
-The Settings custom resource should look like this:
+To use our custom auth server, we need to edit the Gloo Settings resource. Run the following command to edit the settings 
+directly in your terminal window:
```

Comment:
i believe this defaults to the program in the `EDITOR` env var, which isn't necessary in your terminal